src/System.Linq.Expressions/src/System/Runtime/CompilerServices/CallSite.cs

```diff
@@ -277,6 +277,47 @@ internal void MoveRule(int i)
             }
         }
 
+        // TODO https://github.com/mono/linker/issues/799: Consolidate these attributes when possible.
```

Comment:
`#if FEATURE_COMPILE` to match the implementation?

Reply:
Yup. Will add.